test/integration.ts

```diff
@@ -2,33 +2,47 @@
 const SignerProvider = require('ethjs-provider-signer');
 const { sign } = require('ethjs-signer');
 import assert from 'assert';
-const fs = require('fs');
 const rpcUrl = process.env['ETH_RPC_URL'];
 // tslint:disable-next-line:import-name
 import BN from 'bn.js';
+// tslint:disable-next-line:import-name
+import contractAddresses from './addresses_tinlake.json';
 
 import Tinlake from '../dist/Tinlake';
 
 const SUCCESS_STATUS = '0x1';
 
-let tinlake: Tinlake;
-const ethFrom = `0x${process.env['ETH_FROM']}`;
+let adminTinlake: Tinlake;
+let borrowerTinlake: Tinlake;
+const adminEthFrom = `0x${process.env['ETH_FROM']}`;
```

Comment:
Can we have some kind of defaults for the envs in order for the tests to un in travis?

Reply:
I sent you the env variables to use in Travis